packages/features/bookings/Booker/store.ts

```diff
@@ -0,0 +1,154 @@
+import { useEffect } from "react";
+import { create } from "zustand";
+
```

Comment:
Booker store that contains all shared booker data and data manipulation methods that are used by it's child components.  See interface comment for more details.